client/src/components/Geolocation.jsx

```diff
@@ -0,0 +1,79 @@
+import { useEffect, useState } from "react";
+import {
+  MapContainer,
+  TileLayer,
+  Marker,
+  Popup,
+  ZoomControl,
+} from "react-leaflet";
+import MarkerClusterGroup from "react-leaflet-cluster";
+import "leaflet/dist/leaflet.css";
+import { Icon, divIcon, point } from "leaflet";
+import connexion from "../services/connexion";
+import LocationMarker from "./LocationMarker";
+import "../styles/Geolocation.css";
+import "../App.css";
+
+export default function Geolocation() {
+  const [streetArts, setStreetArts] = useState([]);
+  useEffect(() => {
+    connexion
+      .get("api/streetarts")
+      .then((response) => {
+        setStreetArts(response.data);
+      })
+      .catch((error) => {
+        console.error("There war an error fetching the users!", error);
```

Comment:
There is an error fething the USER ??? ne cherche tu pas à récupérer les streetarts ?